### ISSUE TITLE
transmission-cli -w or transmission-cli --download-dir are completely ignored

### ISSUE THREAD

**Issue by gnarlin** [transmission/transmission#1720]
I try to download with transmission -w path or --download-dir path and yet transmission-cli ALWAYS downloads only to ~/Downloads.

**Comment by tearfur** [comment]
`transmission-remote localhost:9093 -a https://releases.ubuntu.com/22.04/ubuntu-22.04.3-live-server-amd64.iso.torrent -w ~` works fine for me.

**Comment by tearfur** [comment]
Oops this issue is about `transmission-cli` not `transmission-remote`.

**Comment by tobbez** [comment]
I think this can be closed. `--download-dir`/`-w` appears to work.

I submitted a fix for a bug with the same symptoms (#6920), but that bug was introduced after this issue was filed.

There is one other cause of `--download-dir` being ignored: if transmission-cli is launched *without* the option, stopped, and then launched *with* the option, it will be ignored (as the path from the resume state is used instead):

```sh
$ transmission-cli example.torrent # no --download-dir
^C
$ transmission-cli --download-dir . example.torrent # --download-dir ignored, path in resume state takes precedence
```
